exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/JaegerGrpcSpanExporter.java

```diff
@@ -126,6 +126,7 @@ public ResultCode export(Collection<SpanData> spans) {
       stub.postSpans(request);
       return ResultCode.SUCCESS;
     } catch (Throwable e) {
+      logger.log(Level.SEVERE, "Failed to export spans", e);
```

Comment:
We don't seem to use `SEVERE` anywhere else yet - spans can fail to export for transient reasons (backend is restarting), and observability is not critical to the app anyways, so I'd go with WARNING if not INFO.

Reply:
Please at least WARN :)

Reply:
WARN works for me. Will update.